data/scripts/sol-flow-node-type-gen.py.in

```diff
@@ -258,6 +258,7 @@ data_type_to_c_map = {
     "direction-vector": "struct sol_direction_vector",
     "string": "const char *",
     "location": "struct sol_location",
+    "timestamp": "time_t",
```

Comment:
ok about timezone, I still wonder about the precision.

If we change from `time_t` to `struct timespec`, then we could carry the extra precision in nanoseconds and use the `sol_util` functions to retrieve clock in such format.

regarding format to string, we can handle it ourselves by calling `strptime()` or `strftime()` in the seconds part, then doing the nanoseconds manually.

JSON shouldn't matter, it keeps doing the low-precision version and gives `timespec.tv_nsec = 0`.